mllib/src/main/scala/org/apache/spark/ml/feature/LabeledPoint.scala

```diff
@@ -35,4 +35,11 @@ case class LabeledPoint(@Since("2.0.0") label: Double, @Since("2.0.0") features:
   override def toString: String = {
     s"($label,$features)"
   }
+
+  private[spark] def toInstance: Instance = toInstance(1.0)
```

Comment:
Actually, I'd prefer to remove the no arg function and be explicit everywhere. That way there is no ambiguity or unintended effects if someone changes the default value. Sound ok?